### PR TITLE
Small documentation change to Get-AzLog.md

### PR DESCRIPTION
**Pull request by theheatDK**
The -CorrelationId parameter is not a required parameter.

It is not mentioned anywhere that the module retrieves Activity Log events. It is nice with a bit of context. In the Portal it is called Activity Log and the CLI uses "az monitor activity-log".

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Monitor/Monitor/help/Get-AzLog.md [hunk 1]
```diff
@@ -213,7 +213,6 @@ Accept wildcard characters: False
 
 ### -CorrelationId
 Specifies the correlation ID.
-This parameter is required.
```
Comment (VeryEarly) [inline src/Monitor/Monitor/help/Get-AzLog.md]: this parameter seems required
[Mandatory = true](https://github.com/Azure/azure-powershell/blob/master/src/Monitor/Monitor/ActivityLogs/GetAzureRmLogCommand.cs#L68)

**Comment by VeryEarly** [comment]
It's true that the name is inconsistent with CLI and azure portal. 
Cmdlet name cannot be changed until next breaking change release, we can add alias to it.

File: src/Monitor/Monitor/help/Get-AzLog.md (hunk 1 above)
Comment (theheatDK) [reply src/Monitor/Monitor/help/Get-AzLog.md]: But it is not. Look at the examples.

Comment (VeryEarly) [reply src/Monitor/Monitor/help/Get-AzLog.md]: Can you point which example indicates "-CorrelationId" is not required?

Comment (theheatDK) [reply src/Monitor/Monitor/help/Get-AzLog.md]: Get-AzLog -MaxRecord 100
Get-AzLog -ResourceId $ResourceId -MaxRecord 5
Get-AzLog -DetailedOutput -ResourceId $ResourceId -Caller '*@dnhf.onmicrosoft.com'

All of these works fine.

Comment (VeryEarly) [reply src/Monitor/Monitor/help/Get-AzLog.md]: cmdlets can have multiple parameter sets,
CorrelationId is required in parameter set "CorrelationIdParameterSetName", other parameter sets could work fine without it.

Comment (theheatDK) [reply src/Monitor/Monitor/help/Get-AzLog.md]: An alias would be nice :-)

Comment (VeryEarly) [reply src/Monitor/Monitor/help/Get-AzLog.md]: I'll close this PR, can you raise another issue regarding the cmdlet name so we can track it? Simply specify the command name in CLI / Portal / desired alias would be enough, thanks.

Comment (theheatDK) [reply src/Monitor/Monitor/help/Get-AzLog.md]: Ok. But when I look at the documentation as it is now:
https://docs.microsoft.com/en-us/powershell/module/az.monitor/get-azlog?view=azps-3.4.0
Then try to look for the word required. It is only mentioned once on the page. 
That does not make any sense to me?

Comment (theheatDK) [reply src/Monitor/Monitor/help/Get-AzLog.md]: Ok, this is the last time I will ever make any contribution to the Azure Powershell module.
I waste my weekend trying to make it better and it is just closed!! No wonder the documentation is so shit.

Comment (VeryEarly) [reply src/Monitor/Monitor/help/Get-AzLog.md]: Hi @theheatDK , 
this PR is to make change the description of parameter "-CorrelationId" which is incorrect as we discussed.

We appreciate your passion and effort to help make azure-powershell better, however to have an alias for this cmdlet, we need a  dedicated issue to track it.

About this help doc, it seems it was not generated properly, thanks for point it out, we'll regenerate doc for monitor in the future releases.

Comment (theheatDK) [reply src/Monitor/Monitor/help/Get-AzLog.md]: I wrote this:

> The -CorrelationId parameter is not a required parameter.
> 
> It is not mentioned anywhere that the module retrieves Activity Log events. It is nice with a bit of context. In the Portal it is called Activity Log and the CLI uses "az monitor activity-log".

I made 3 changes to the text. One regarding the parameter and two to make it clear that this cmdlet is used for Activity logs. I would have assumed that you looked at my changes before dismissing them.

Comment (VeryEarly) [reply src/Monitor/Monitor/help/Get-AzLog.md]: Reopened PR,
Can you add the "This parameter is required." back, and something more meaningful in the ChangeLog.md? Something like "fix description of cmdlet, thanks.

Comment (theheatDK) [reply src/Monitor/Monitor/help/Get-AzLog.md]: Hi @VeryEarly,
I have added the text back and changed the text in the ChangeLog.md.
Regards Peter

**Pull request merged**